hack/test-cmd.sh

```diff
@@ -158,7 +158,7 @@ echo "images: ok"
 osc get imageRepositories
 osc create -f test/integration/fixtures/test-image-repository.json
 [ -z "$(osc get imageRepositories test -t "{{.status.dockerImageRepository}}")" ]
-osc apply -f examples/sample-app/docker-registry-config.json
+osc create -f examples/sample-app/docker-registry-config.json
```

Comment:
yeah this needs rebase
On Feb 5, 2015 5:30 PM, "Ben Parees" <notifications@github.com> wrote:

> In hack/test-cmd.sh
> <https://github.com/openshift/origin/pull/909#discussion_r24175561>:
>
> > @@ -158,7 +158,7 @@ echo "images: ok"
> >  osc get imageRepositories
> >  osc create -f test/integration/fixtures/test-image-repository.json
> >  [ -z "$(osc get imageRepositories test -t "{{.status.dockerImageRepository}}")" ]
> > -osc apply -f examples/sample-app/docker-registry-config.json
> > +osc create -f examples/sample-app/docker-registry-config.json
>
> i think you've removed this file, no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/909/files#r24175561>.
>